ndex2/client.py

```diff
@@ -482,9 +598,32 @@ def get_neighborhood_as_cx_stream(self, network_id, search_string, search_depth=
 
     def get_neighborhood(self, network_id, search_string, search_depth=1, edge_limit=2500):
         """
+
         Get the CX for a subnetwork of the network specified by UUID network_id and a traversal of search_depth steps
         around the nodes found by search_string.
 
+        .. code-block:: python
+
+		from ndex2.nice_cx_network import NiceCXNetwork
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		# Query the network from my NDEx account, get small subset
+		query_result_cx=my_ndex.get_neighborhood('INSERT NETWORK UUID HERE','INSERT NODE NAME HERE')
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client